crates/autopilot/src/solvable_orders.rs

```diff
@@ -337,6 +340,71 @@ impl SolvableOrdersCache {
             .collect()
     }
 
+    /// Returns current solvable orders along with the latest order creation
+    /// timestamp.
+    async fn get_solvable_orders(&self) -> Result<(SolvableOrders, DateTime<Utc>)> {
```

Comment:
Getting the latest order creation timestamp can be done with the `SolvableOrders`. I believe it shouldn't be the responsibility of this function.

Reply:
This function has to return either `previous_creation_timestamp` or `latest_creation_timestamp`. Since the former is not used anywhere else, I decided to keep it in this function.